proposals/0000-template.rst

```diff
@@ -52,6 +52,9 @@ may include,
 * how the proposed change interacts with existing language or compiler
   features, in case that is otherwise ambiguous
 
+The ideal specification is described as a modification of the
+`Haskell 2010 report <https://www.haskell.org/definition/haskell2010.pdf>`_.
```

Comment:
Yes, but I think this is too high a bar. In particular, a good deal of proposals (maybe the majority) are amendments to existing extensions, which are usually not very finely documented.

So maybe this suggests:

```suggestion
The ideal specification is described as a modification of the
`Haskell 2010 report <https://www.haskell.org/definition/haskell2010.pdf>`_.
However, that is not always possible, if the feature you are building from is not part of Haskell 2010 or if you cannot see how to modify the report to include your feature.
```

Reply:
That was the meaning that I intended of the word “ideal”. I'm not sure we should qualify this sentence too much, lest it be unconsciously ignored.

Reply:
How about this?

"Strive for *precision*.   The ideal specification is described as a modification of the Haskell 2010 report. Where that is not possible (e.g. because the specification relates to a feature that is not in the Haskell 2010 report), try to adhere its style and level of detail.   Think about corner cases.   Write down general rules and invariants.  "